From d630dee374abb4e530250d7e573447b24d466509 Mon Sep 17 00:00:00 2001
From: Cesare Mercurio <cesare.mercurio@gmail.com>
Date: Sat, 16 Dec 2023 13:26:53 -0800
Subject: [PATCH] VVI-flatbuffers-patch : Fix build bazel for npm packages

Summary:
Removes the dependency to npm for c++ only environment

Test Plan:
integration in vvplatform
---
 BUILD.bazel | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/BUILD.bazel b/BUILD.bazel
index b4f015a0..8cf7a55e 100644
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -1,5 +1,3 @@
-load("@aspect_rules_js//npm:defs.bzl", "npm_link_package")
-load("@npm//:defs.bzl", "npm_link_all_packages")
 load("@rules_cc//cc:defs.bzl", "cc_binary", "cc_library")
 
 licenses(["notice"])
@@ -8,13 +6,6 @@ package(
     default_visibility = ["//visibility:public"],
 )
 
-npm_link_all_packages(name = "node_modules")
-
-npm_link_package(
-    name = "node_modules/flatbuffers",
-    src = "//ts:flatbuffers",
-)
-
 exports_files([
     "LICENSE",
     "tsconfig.json",
-- 
2.34.1

